repository/ConfigurationOfOSSubprocess.package/ConfigurationOfOSSubprocess.class/instance/stable..st

```diff
@@ -2,4 +2,6 @@ symbolic versions
 stable: spec
 	<symbolicVersion: #stable>
 	
+	spec for: #'common' version: '0.2.4'.
```

Comment:
Why add a #common section? This would allow OSSubprocess to try to load on all Pharo versions in existence (for example 1.3, 2.0, 3.0, 4.0) with the results we would expect...